### DISCUSSION TITLE
alist是否支持除了视频文件以外的文件下载

### DISCUSSION THREAD

**Discussion by oldbird007** [AlistGo/alist#6677] [General]
我用的是windows系统的v3.35.0版本，下载视频类的文件都是正常的，但是下载zip、exe、pdf文件都是无法使用的，下面是我的配置信息
![微信截图_20240701160334](https://github.com/alist-org/alist/assets/124140317/ff4bcf9f-3123-4f7e-96dd-9ab920302eb4)
![微信截图_20240701160436](https://github.com/alist-org/alist/assets/124140317/3d8011d2-bc51-4015-a92c-8d56e9ea4905)
我不知道是我配置的有问题，还是alist只支持视频文件的下载

**Comment by pongfcnkl** [comment]
下载接口选非官方只能下视频
你可以选官方接口

**Comment by oldbird007** [comment]
谢谢了，官方确实可以用那个速度也很官方
